lib/api/apiUtils/bucket/bucketCreation.js

```diff
@@ -172,9 +173,20 @@ function createBucket(authInfo, bucketName, headers,
         BucketInfo.currentModelVersion(), null, null, null,
         null, null, null, null, null, null, null, null,
         null, isNFSEnabled);
+    let locationConstraintVal = null;
 
-    if (locationConstraint !== undefined) {
-        bucket.setLocationConstraint(locationConstraint);
+    if (locationConstraint) {
+        const locationConstraintStr =
+            locationConstraint.split(zenkoSeparator)[0];
```

Comment:
Just wondering, do we check that the location constraint name does not contain a zenkoSeparator (`:`). e.g. if `locationConstraint =  's3c:test1'`

Reply:
At least from Orbit, when adding a storage location users are not allowed to have ':' in the name